test/toTex_test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+
+const parse = require('../docs/bundle.js');
+const toTex = require('../lib/toTex.js');
```

Comment:
this is the line that's causing the issue, try `const {parse, toTex} = require('../docs/bundle.js');`